RetakesPlugin/Modules/Managers/GameManager.cs

```diff
@@ -198,6 +195,28 @@ public void BalanceTeams()
         SetTeams(newTerrorists, newCounterTerrorists);
     }
 
+    public void OnRoundPreStart(CsTeam winningTeam) {
+        // Handle team swaps during round pre-start.
+        switch (winningTeam)
```

Comment:
Can we also keep the calls to the debug logs too

Reply:
There where no calls to Helpers.WriteLine within the GameManager before? Do you want me to add granular logs inside GameManager or is it enough to have debug logs in RetakesPlugin.cs around the call to OnRoundPreStart?

By the way it seems the LogPrefix is added twice for the debug log as each call to Helpers.WriteLine adds the prefix by the caller but then the prefix is also added inside the method.

Reply:
For now we can just port the existing logs that were in the part of the `RetakesPlugin.cs` that was migrated over. The helper is statically called so it shouldn't be an issue. Most of the helpers that were added were to aid stability in the earlier development of the plugin. Don't worry about adding extras in unless it's not behaving as expected.

Reply:
Ok I added the debug prints arround the call to GameManager.OnRoundPreStart now before I saw your reply? Should I move it inside arround the calls to CounterTerroristRoundWin etc?

Reply:
By the way I will make a parallel PR on master to remove the double prefix.